### DISCUSSION TITLE
What's the difference with mdbook

### DISCUSSION THREAD

**Discussion by shenjackyuanjie** [web-infra-dev/rspress#331] [Q&A] [answered]
虽说大概看了一下文档（
但我还是想问，跟mdbook (rust-lang/mdbook) 有什么不同之处呢（

**Comment by shenjackyuanjie** [comment]
以及可以在文档里提到mdbook，毕竟都是rust based

**Comment by sanyuan0704** [comment, marked as answer]
Rspress 主要差异点如下：

1. 支持 mdx，对前端开发者友好，可以嵌入 React 组件
2. 支持模块文档 API 生成(JSDoc)
3. 支持组件的 preview 预览及组件 playground 实时编辑

**Reply by shenjackyuanjie** [reply]
那可以在文档里提到呀，（也能顺便从mdbook那里吸引一些人）（我是从rspack看到的，如果能在mdbook相关看到应该会更早接触）
以及现在没有计划做一个类似tarui的从`cargo`的入口吗？

**Reply by sanyuan0704** [reply]
Rspress 本质上是一个 npm 包，不提供 Rust 侧的入口，其中用到的 Rust 代码只是为了取代原有的一些编译步骤
